docker/config_template.yaml

```diff
@@ -197,13 +197,25 @@ clusterMetadata:
     enableGlobalDomain: {{ default .Env.ENABLE_GLOBAL_DOMAIN "false" }}
     failoverVersionIncrement: 10
     masterClusterName: "master"
+    {{- if .Env.IS_NOT_MASTER }}
```

Comment:
For some reason, let's not use the word master. What about is_not_active? And if you can replace the master with other word in this file. It would be great.